fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/SplitHelper.java

```diff
@@ -788,6 +808,13 @@ public boolean accept(@Nonnull RecordCursorVisitor visitor) {
                         if (next == null) {
                             continuation = innerResult.getContinuation();
                         }
+                        if (LOGGER.isTraceEnabled()) {
```

Comment:
Does it make sense to emit some StoreTimer info into the log message? If it really is taking over 10 seconds to unspilt a record perhaps we might want more detail into what is going. E.g. here when the inner cursor stops.

Reply:
Hm, I thought it over, and I don't think so, but I could be wrong. My main concern is that that could pollute the information here with statistics from other operations, whereas this is supposed to trace through the unsplitter logic to see if something within it is tripping it up. But along those lines, I thought that adding a "time since last read" metric and logging it with appends might be good to have.

Reply:
I added that in a separate commit.